demoapp/www/js/crashes.js

```diff
@@ -119,15 +112,19 @@ $(document).bind('pageinit', function () {
         LowMemory.generateLowMemory();
     });
 
-    var updateAttachment = function () {
-        $('#text_attachment_value').html("Current value: " + text);
-        $('#file_attachment_value').html("Current value: " + attachment);
+    var updateAttachmentUI = function () {
+        $('#text_attachment_value').text("Current value: " + textAttachment);
+        var imageDesc = null;
+        if (typeof (binaryAttachment) === "string") {
```

Comment:
Can it be not `string`? Is it null if we have no image stored? Should we then print `No image attached`?

Reply:
Btw if `binaryAttachemnt` contains teh uri why not just print it or some part of it?

Reply:
This is base64 data and this check is to avoid checking for both null and undefined while being strict about the type.